Jenkinsfile

```diff
@@ -20,6 +20,7 @@ pipeline {
                     set -exo pipefail
                     # Ensures the currently released Docker Python image is used.
                     docker pull gcr.io/kaggle-images/python:staging
+		    docker pull gcr.io/kaggle-images/python@sha256:287c4e0e224e592dc6113940a6cf3d099b814c7bff0c1e8da57f8e6bad123ac5
```

Comment:
Can this be `python:v143`?  Also, consider a comment here, since it may not be clear why we need 2 versions of the same image.

Reply:
i gave up looking for this thanks!